README.md

```diff
@@ -56,6 +56,12 @@ If you want to report a bug or create a feature-request, you can open a new issu
 [GitHub](https://github.com/mumble-voip/mumble/issues/new/choose).
 
 
+# Code Signing
```

Comment:
This should be a `##` (`h2` headline), not `#` (`h1`, or main headline) as it breaks the headline semantics.

An `h1` should be the main headline and only be used once in a document.

As it is currently, it feels that `Windows`, `MacOS` and everything following are sub-parts of `Code Signing`.

![image](https://github.com/mumble-voip/mumble/assets/2989985/df688589-7086-49d4-9f2f-a0c172e0692a)

Reply:
A generated Table of Contents from the current document structure shows this pretty clearly:

![image](https://github.com/mumble-voip/mumble/assets/2989985/28e7906b-433a-4b03-874b-7a39e295510e)

Reply:
Indeed - that's what you get for copy-pasting xD

Reply:
Thanks for bringing this up!